src/vivarium_cluster_tools/psimulate/distributed_worker.py

```diff
@@ -206,13 +182,14 @@ def worker(parameters: Mapping):
         event["end"] = time()
         end_snapshot = CounterSnapshot()
 
-        do_sim_epilogue(start_snapshot, end_snapshot, event, exec_time, parameters)
+        do_sim_epilogue(start_snapshot, end_snapshot, event, exec_time, job_parameters)
 
         idx = pd.MultiIndex.from_tuples(
-            [(input_draw, random_seed)], names=["input_draw_number", "random_seed"]
+            [(job_parameters.input_draw, job_parameters.random_seed)],
+            names=["input_draw_number", "random_seed"],
         )
         output_metrics = pd.DataFrame(metrics, index=idx)
-        for k, v in collapse_nested_dict(run_key):
+        for k, v in collapse_nested_dict(job_parameters.branch_configuration):
```

Comment:
This will break results processing, which has to drop these columns, but that should be a one line fix.